streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -272,6 +275,7 @@ public void shouldGetAllFromCache() {
         assertFalse(iterator.hasNext());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
It actually seems like using the non-deprecated `put` method would improve this test's readability, since the verification depends on specific timestamps having been used in the put.